src/container-manager.ts

```diff
@@ -65,6 +77,13 @@ export class DockerContainerManager implements ContainerManager {
 export class PodmanContainerManager implements ContainerManager {
   constructor(private command: string) {}
 
+  getImagePlatform(imageName: string): ImagePlatformResult {
+    const getPlatform = spawnSync(this.command, ['image', 'inspect', '--format', '{{.Os}} {{.Architecture}}', imageName])
```

Comment:
Haven't tested podman, but [the docs](https://docs.podman.io/en/latest/markdown/podman-inspect.1.html) indicate that the `inspect` command behaves the same as it does in docker.

Reply:
I kinda want to rip out support for podman, I don't think anybody using jamsocket uses it.

Reply:
Yeah, me too.

Reply:
https://github.com/drifting-in-space/jamsocket-cli/pull/33